pystac/summaries.py

```diff
@@ -25,6 +26,12 @@
     from pystac.collection import Collection
     from pystac.item import Item
 
+here = Path(__file__).parent.resolve()
+
+# Every time pystac is released this file gets pulled from
+# https://cdn.jsdelivr.net/npm/@radiantearth/stac-fields/fields-normalized.json
+FIELDS_JSON_PATH = Path(here, "static/fields-normalized.json")
```

Comment:
I _think_ it's safer to use `importlib_resources`, e.g. https://github.com/stactools-packages/noaa-cdr/blob/7dc626225e78f701bc723e33b88f1e5f4b455d7b/src/stactools/noaa_cdr/ocean_heat_content/constants.py#L40-L44.

Reply:
OMGGGG I did not know about that. The main difference in that approach is that in the dev version you would get the latest verision aka the file would not live in git. But that seems like a fine compromise for a much more standardized approach.

Reply:
oh wait I misunderstood. I thought this was something that you could just put in the setupy.py and have it pulled into the package data. Ignore me.